### ISSUE TITLE
Stroke styling: size / color

### ISSUE THREAD

**Issue by ivanmalagon** [CartoDB/cartodb#13906]
### Context

Project [Split size/color and stroke components](https://github.com/CartoDB/product/issues/54)

Design ticket: https://github.com/CartoDB/design/issues/1082

Invision project: https://projects.invisionapp.com/share/D9FDRB6384S

### Notes

Don't merge this issue until all the three geometries issues are finished.

### Goal

The goal of this project is to refactor the styling UX to make it easier to use. In the attached project you can find all the information and context.

The screenshots says `Points` but we're referring here to `Stroke`

#### Stroke size - fixed

![point size fixed](https://user-images.githubusercontent.com/14546701/35054413-c9240aa8-fbac-11e7-9de2-59dd4508d7f8.png)

#### Stroke size - by value

![point size by value](https://user-images.githubusercontent.com/14546701/35054558-1f637e4e-fbad-11e7-8929-f38618d8de26.png)

When you select a column, it must show the dialog to select buckets, quantiles and palette.

![image](https://user-images.githubusercontent.com/1078228/39355550-d03980fc-4a0d-11e8-8f8c-4ae5421844f6.png)

#### Stroke color - solid

![point color solid](https://user-images.githubusercontent.com/14546701/35054492-f6760664-fbac-11e7-808b-af306e71dd18.png)

#### Stroke color - by value

Text column

![point color by value](https://user-images.githubusercontent.com/14546701/35054499-fc3c09ae-fbac-11e7-9b02-472e1056a2d8.png)

Numeric column

The dialog changes from the text onee

![image](https://user-images.githubusercontent.com/1078228/39355089-891a7a92-4a0c-11e8-806d-ac9e78d7cf99.png)

![image](https://user-images.githubusercontent.com/1078228/39355695-30161648-4a0e-11e8-9a20-ac7cbd939b84.png)

**Comment by piensaenpixel** [comment]
One small tweak 
- When we have a line layer - We should put 'stroke color' instead of  'stroke'

**Comment by d-damien** [comment]
You'd need to split size / color / shape. Also, why has this disappeared in more recent versions ?

Should I open a new issue about these topics ?
